cfme/utils/appliance/implementations/ui.py

```diff
@@ -54,6 +54,18 @@ def get_rails_error(self):
 
 class MiqBrowserPlugin(DefaultPlugin):
     ENSURE_PAGE_SAFE = jsmin('''\
+        try {
```

Comment:
Could we leave a little comment here explaining how problematic these notifications are?